src/cli.ml

```diff
@@ -0,0 +1,101 @@
+let file_cmd =
+  let doc = "A wikicreole file to convert to HTML." in
+  Cmdliner.Arg.(non_empty & pos_all file [] & info [] ~docv:"FILE" ~doc)
+
+let print_cmd =
+  let doc = "Print the HTML to stdout." in
+  Cmdliner.Arg.(value & flag & info ["p"; "print"] ~doc)
+
+let outfile_cmd =
+  let doc = "Writes the HTML into the given file. Always overwrites.
+Overrides $(b,--print)." in
+  Cmdliner.Arg.(value & opt (some string) None & info ["o"; "output"]
+                  ~docv:"FILE" ~doc)
+
+let cwd = Sys.getcwd ()
+let root_cmd =
+  let doc = "Use the given root directory." in
+  Cmdliner.Arg.(value & opt string cwd & info ["root"]
+                  ~docv:"DIR" ~doc)
+
+let manual_cmd =
+  let doc = "Use the given manual path." in
+  Cmdliner.Arg.(value & opt string cwd & info ["manual"]
+                  ~docv:"DIR" ~doc)
+
+let api_cmd =
+  let doc = "Use the given api path." in
+  Cmdliner.Arg.(value & opt string cwd & info ["api"]
+                  ~docv:"DIR" ~doc)
+
+let info_cmd = Cmdliner.(
+    let doc = "Converts a wikicreole file into an HTML file." in
+    let man = [
+      `S Manpage.s_description;
+      `P "$(tname) is a command line utility for compiling Wikicreole files
+into HTML. A description of its syntax can be found at
+<http://ocsigen-1.inria.fr/ocsimore/dev/manual/wiki>.";
+
+      `S "EXTENSIONS";
```

Comment:
Do you think it is the good place to document the extensions?
This will probably have to be documented on the Web site.
We must avoid duplication of documentation.

Reply:
Probably not but in the meantime I think it is good to have it written somewhere.